_pages/en/technical-platform.md

```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
```

Comment:
Spelling mistake here?
Is mobile-friendly really the qualifier we want to put here?

Reply:
fixed